### ISSUE TITLE
PyQt5 installs some files in Python and Qt5 install directories

### ISSUE THREAD

**Issue by guest1604** [easybuilders/easybuild-easyconfigs#7409]
PyQt5 installs some files in Python and Qt5 install directories. eg for "PyQt5-5.11.3-foss-2018b-Python-3.6.6.eb" :
- software/MPI/GCC/7.3.0-2.30/OpenMPI/3.1.1/Python/3.6.6/bin/pylupdate5
- software/MPI/GCC/7.3.0-2.30/OpenMPI/3.1.1/Python/3.6.6/bin/pyrcc5
- software/MPI/GCC/7.3.0-2.30/OpenMPI/3.1.1/Python/3.6.6/bin/pyuic5
- software/MPI/GCC/7.3.0-2.30/OpenMPI/3.1.1/Qt5/5.10.1/plugins/PyQt5/libpyqt5qmlplugin.so
- software/MPI/GCC/7.3.0-2.30/OpenMPI/3.1.1/Qt5/5.10.1/plugins/designer/libpyqt5.so

So we can't install several versions of PyQt5 in the same toolchain.

Do you know if this is what is expected ?

**Comment by boegel** [comment]
@guest1604 That's definitely not intentional, installations are supported to be self-contained...

Apparently the `--destdir` option is not sufficient, and the location for binaries and plugins need to be specified separately?

Can you look into what the correct option are for that?

**Comment by guest1604** [comment]
@boegel Thanks for your answer, I look the correct options Monday.

**Comment by guest1604** [comment]
Using `--bindir`, `--designer-plugindir` and `--qml-plugindir` configure options solve this issue :

```
'configopts': "configure.py --confirm-license --verbose --destdir=%s --no-sip-files --no-stubs --bindir=%%(installdir)s/bin --designer-plugindir=%%(installdir)s/plugins/designer --qml-plugindir=%%(installdir)s/plugins/PyQt5" % pylibdir,
```

I don’t know if it’s relevant to configure QT_PLUGIN_PATH variable :

```
modextrapaths = {
    'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages',
    'QT_PLUGIN_PATH': 'plugins',
}
```

**Comment by JackPerdue** [comment]
I see that problem with a number of Python packages.  I recommend using EASYBUILD_READ_ONLY_INSTALLDIR=1  or  --read-only-installdir when installing Python (actually, when installing anything) to avoid this kind of pollution.  Doesn't solve the problem (since the build will likely fail) but at least alerts you to it.

**Comment by boegel** [comment]
should be fixed by #7470...

**Comment by boegel** [comment]
closing this since #7470 is now merged (to be included with EasyBuild v3.8.1)
